Makefile

```diff
@@ -55,7 +55,7 @@ BUILDTAGS ?= \
 	$(shell hack/btrfs_tag.sh) \
 	$(shell hack/systemd_tag.sh) \
 	$(shell hack/libsubid_tag.sh) \
-	seccomp
+	$(if $(filter Linux,$(shell uname -a)), seccomp,)
```

Comment:
I think this should use `uname -s` which is what the other uname conditions in the Makefile are currently using.
Or is there a Linux distro where `uname -s` does not report `Linux`?

Reply:
Agreed; the way it now may even result in a false positive (for example, if someone will use Linux in the hostname on their non-linux box).

OTOH it's even better to use GOOS or NATIVE_GOOS since it's already defined (and used in some places).

Updated; PTAL